netbox/extras/middleware.py

```diff
@@ -35,7 +39,7 @@ def mark_object_changed(instance, **kwargs):
     _thread_locals.changed_objects.append((instance, action))
 
 
-class ChangeLoggingMiddleware(object):
+class ChangeLoggingAndWebhookMiddleware(object):
```

Comment:
Not a fan of this name. If we're going to combine functions, let's just call it something like `ObjectChangeMiddleware` in case we extend it in the future.